.github/workflows/nightly-test.yaml

```diff
@@ -14,58 +19,86 @@ jobs:
       matrix:
         os: ["ubuntu:20.04", "ubuntu:22.04", "ubuntu:24.04"]
         arch: ["amd64", "arm64"]
-        release: ["latest/edge"]
+        channel: ["latest/edge"]
```

Comment:
We should probably test our release channels as well for the nightly test and add `1.32-classic/stable`.

Reply:
We don't need that.
We run the full test suite before populating/backporting to `release-1.XX` branches. This would just run the same tests again.
So, we only need `latest/edge`

Reply:
@bschimke95 thanks for the clarification.